layouts/partials/navbar.html

```diff
@@ -1,7 +1,7 @@
 {{ $cover := .HasShortcode "blocks/cover" }}
 <nav class="js-navbar-scroll navbar navbar-expand navbar-dark {{ if $cover}} td-navbar-cover {{ end }}flex-column flex-md-row td-navbar">
 	<a class="navbar-brand" href="{{ .Site.Home.RelPermalink }}">
-		<span class="navbar-logo">{{ if .Site.Params.ui.navbar_logo }}{{ with resources.Get "icons/logo.svg" }}{{ ( . | minify).Content | safeHTML }}{{ end }}{{ end }}</span><span class="text-uppercase font-weight-bold">{{ .Site.Title }}</span>
+		<span class="navbar-logo">{{ if (fileExists "icons/logo.svg") }}{{ with resources.Get "icons/logo.svg" }}{{ ( . | minify).Content | safeHTML }}{{ end }}{{ end }}</span><span class="text-uppercase font-weight-bold">{{ .Site.Title }}</span>
```

Comment:
This would not work for me unless I added the `/assets/` path:

```suggestion
		<span class="navbar-logo">{{ if (fileExists "assets/icons/logo.svg") }}{{ with resources.Get "icons/logo.svg" }}{{ ( . | minify).Content | safeHTML }}{{ end }}{{ end }}</span><span class="text-uppercase font-weight-bold">{{ .Site.Title }}</span>
```

Reply:
That's weird because I didn't change those in this commit, that's the existing navbar code in the repo and it's working fine in both goldydocs and the Docsy guide. Let me check it again....

Reply:
Oh hang on, I didn't mean to change that in this commit, this is what happens when you have old changes knocking around a branch, that's actually old code. I'll revert.

(it must have worked for me locally, hence I didn't notice....)

Reply:
Fixed!